fitting/fitScanData.py

```diff
@@ -276,11 +276,11 @@ def fit(self, debug=False):
                     #if (self.calDAC2Q_m[vfat]*(rand)+self.calDAC2Q_b[vfat]) < 0: continue
 
                     # Provide an initial guess
-                    init_guess_p0 = self.calDAC2Q_m[vfat]*(8+stepN*8)+self.calDAC2Q_b[vfat]
+                    init_guess_p0 = self.calDAC2Q_m[vfat]*(8+stepN*8)+self.calDAC2Q_b[vfat] 
                     if self.isVFAT3:
                         init_guess_p1 = self.calDAC2Q_m[vfat]*(256)+self.calDAC2Q_b[vfat]+abs(self.calDAC2Q_m[vfat]*rand)  #self.calDAC2Q_m[vfat] might be negative (e.g. VFAT3 case)
-		    else:
-			init_guess_p1 = abs(self.calDAC2Q_m[vfat]*rand)
+                    else:
+                        init_guess_p1 = abs(self.calDAC2Q_m[vfat]*rand)
```

Comment:
It looks like you've inserted the `\t` character here?  Maybe git is just rendering it as a tab for some reason.

If it is a tab, it should be replaced by four `\s` (space) characters since different editors may render the `\t` character differently and in a whitespace sensitive language (e.g. python) this could cause conflicts (e.g. case statements not matching up).

To convert tabs to spaces see this [vim example ](http://vim.wikia.com/wiki/Converting_tabs_to_spaces)

Reply:
they way it is now is correct, with no tabs and all spaces (had tabs before)